.github/workflows/conformance-k8s-kind-network-policies.yaml

```
@@ -0,0 +1,259 @@
+name: ConformanceK8sKindNetpol
```

```suggestion
name: K8sUpstreamNetConformance
```
Can we change the name to "Upstream". That's how we refer to the conformance tests in the Cilium project. It will help other community members to understand what this is.

---

sure,